packages/access-client/src/agent.js

```diff
@@ -25,6 +26,23 @@ import {
 
 const HOST = 'https://access.web3.storage'
 
+/** @type {import('./types').Abilities[]} */
+export const abilitiesAsStrings = [
```

Comment:
Aren't _abilities_ always strings?

I think this should be exported from `capabilities` export not here.